src/Extension.Tests/OutputEvaluationTests.cs

```diff
@@ -46,37 +49,73 @@ public async Task output_with_error_event_produces_failed_evaluation()
         }
 
         [Fact]
-        public void when_the_output_passes_all_rules_then_evaluation_passes()
+        public async Task if_teacher_does_not_set_challenge_outcome_it_is_set_as_a_conjunction_of_rule_outcomes_fail_case()
```

Comment:
I know we mentioned that challenge does not have a concept of pass or fail, but I thought it might be useful still have the concept of outcome and the ability to set that outcome.

Previously, these tests tested a default behavior of creating challenge evaluations. But they constructed RuleContext  artificially. So I've modified these tests to use user gestures. These tests document the existing behavior.